layouts/error.vue

```diff
@@ -0,0 +1,40 @@
+<template>
+  <div>
+    <v-row justify="center" class="mt-16 mb-4">
+      <v-icon :color="icon.color" size="72">{{ icon.icon }}</v-icon>
+    </v-row>
+    <v-row justify="center">
+      <h1 class="text-h4">{{ error.message }}</h1>
```

Comment:
Looks good.
I would make this slightly smaller though

Reply:
Switching to h5 is really smaller though, I'll look into other size classes

Reply:
Oh maybe you're talking about the icon?

Reply:
![Screenshot_2020-09-05 jellyfin-vue(1)](https://user-images.githubusercontent.com/1619359/92310077-28223d00-efab-11ea-8a4d-1cc8de8fe835.png)
![Screenshot_2020-09-05 jellyfin-vue](https://user-images.githubusercontent.com/1619359/92310079-28bad380-efab-11ea-9a91-843e6b7c58fc.png)

Reply:
Looks good like this, I think :+1:

Reply:
Actually yeah h4 is too small, h5 looks good to me